ml-agents/mlagents/trainers/learn.py

```diff
@@ -426,15 +426,12 @@ def create_environment_factory(
         #         container.
         # Navigate in docker path and find env_path and copy it.
         env_path = prepare_for_docker_run(docker_target_name, env_path)
-    seed_count = 10000
-    seed_pool = [np.random.randint(0, seed_count) for _ in range(seed_count)]
 
     def create_unity_environment(
         worker_id: int, side_channels: List[SideChannel]
     ) -> UnityEnvironment:
-        env_seed = seed
-        if not env_seed:
```

Comment:
This would only happen if you set `--seed=0` or 1/10000 times if this block
https://github.com/Unity-Technologies/ml-agents/blob/811825b898fe75934a3599bd0856b1ba4a8740a6/ml-agents/mlagents/trainers/learn.py#L492-L494
picked 0 for the seed. 

np.random's seed isn't set at this point, so if you set seed=0 you wouldn't get reproducible results.